OptimizelySDK/Entity/FeatureFlag.cs

```diff
@@ -34,6 +34,13 @@ public List<FeatureVariable> Variables
             }
             set
             {
+                foreach (FeatureVariable variable in value) 
```

Comment:
This is not the good place. I will recommend to add it in `FeatureVariable` class. Use it like this. The major issue is, if `FeatureVariable` class is used outside or by someone, will get different result.
```
private string _subType;
        public string SubType { get {
                return _subType;
            }
            set {
                _subType = value;
            }
        }
        private string _type;
        public string Type {
            get {
                 if (_type == STRING_TYPE && _subType == JSON_TYPE) {
                    return JSON_TYPE;
                }
                return _type;
            }
            set {
                _type = value;
            }
         }
```